### DISCUSSION TITLE
direct line to the destination

### DISCUSSION THREAD

**Discussion by abdullahO2** [organicmaps/organicmaps#2572] [Ideas]
Is it possible to add an optional button to draw a direct line to the destination that is present with the other guiding line, like this:
![image](https://user-images.githubusercontent.com/61907628/168642771-d765d6ea-5115-4648-876f-076b6d588eac.png)
Thank you

**Comment by biodranik** [comment]
We plan to add [helicopter routing](https://github.com/organicmaps/organicmaps/issues?q=is%3Aissue+is%3Aopen+helicopter). Why do you need it?

**Reply by abdullahO2** [reply]
We live in a desert country, and when we need to reach a destination inside the desert, we need multiple routing ways that lead to it, so sometimes the direct line leads to the destination more than the roads drawn on the map and sometimes the opposite, so we need the direct line option to be an additional option for those who want it.
